fhir-server/src/main/java/com/ibm/fhir/server/resources/FHIRResource.java

```diff
@@ -545,7 +545,8 @@ public Response conditionalDelete(@PathParam("type") String type) throws Excepti
             status = e.getHttpStatus();
             return exceptionResponse(e);
         } catch (FHIRPersistenceResourceNotFoundException e) {
-            status = Response.Status.NOT_FOUND;
+            // Return 200 instead of 404 to pass TouchStone test
+            status = Response.Status.OK;
```

Comment:
can we do a 204 instead?   or touchstone is only happy with 200?

Reply:
because we have operation outcome, so I chose to use 200.
Touchstone can accept:
(1) 200 with operation outcome.
(2) 204 not content.